src/librbd/MirroringWatcher.cc

```diff
@@ -74,7 +74,8 @@ void MirroringWatcher<I>::notify_image_updated(
     Context *on_finish) {
 
   CephContext *cct = reinterpret_cast<CephContext*>(io_ctx.cct());
-  ldout(cct, 20) << dendl;
+  ldout(cct, 20) << io_ctx.get_pool_name() << " " << image_id << " "
+                 << global_image_id << " " << mirror_image_state << dendl;
```

Comment:
```suggestion
  ldout(cct, 20) << " pool_name: " << io_ctx.get_pool_name() << "  image_id: " <<  image_id << "  global_image_id: "
                 << global_image_id << " mirror_image_state: " << mirror_image_state << dendl;
```

Reply:
The suggestion to add what is being printed holds for the other methods as well.